File: q1/Questao1.java
class Questao1{
	
	public static void main(String[] args){
		Questao1 q1 = new Questao1();
		System.out.println(q1.resposta());
	}

	int resposta(){
		final double DISTANCIA = 149600000.0; // em km
		final double VELOCIDADE_MEDIA = 28440.0; // em km/h

		Double duracao_viagem = (DISTANCIA / VELOCIDADE_MEDIA); // em horas
		duracao_viagem = Math.ceil((duracao_viagem / 24)); // em dias necessários (teto)

		return duracao_viagem.intValue();
	}	

}

File: q1/Questao1.cs
using System;

namespace CIET
{
    class Questao1
    {

        static double CalcularTempoViagem(double dist_percorrer, double velocidade_km){
            double dias = dist_percorrer / velocidade_km;
            dias = dias / 24;
            return Math.Ceiling(dias);
            
        }

        static void Main(string[] args)
        {
            const double DistPercorrer = 149600000;
            const double VelocidadeKm = 28400;
            double tempo_viagem = CalcularTempoViagem(DistPercorrer , VelocidadeKm);
           

            Console.WriteLine(tempo_viagem);
            
        }
    }
}
